developers/weaviate/concepts/data.md

```diff
@@ -266,6 +266,17 @@ Concretely, a 9-node cluster using `n1-standard-8` machines in our tests could h
 
 Note that these numbers relate to active tenants only. The size of tenants per node can be increased by [setting unused tenants as inactive](../api/rest/schema.md#update-tenants).
 
+### Lazy shard loading
+
+:::info Added in `v1.23`
+:::
+
+When Weaviate starts up, it loads data from all of the shards in your deployment. This process can take a long time. Prior to v1.23, you have to wait until all of the shards are loaded before you can query your data. Since every tenant has it's own shard, multi-tenant deployments can have reduced availability after a restart.
```

Comment:
```
Since every tenant has it's own shard, multi-tenant deployments can have reduced availability after a restart.
```
Propose
```
Since every tenant is a shard, multi-tenant deployments can take a longer time to be ready after a restart.
```

Reply:
Updated